### DISCUSSION TITLE
Just discovered an tool for code reviews called Crocodile

### DISCUSSION THREAD

**Discussion by pirj** [rubocop-hq/rubocop-rspec#1320] [Ideas]
Looks promising, even though the fonts look weird.

https://www.crocodile.dev/

The one feature that caught my attention is that comments don't disappear after changing code and squashing commits.
I also hope that shared links to comments remain, unlike it is with GitHub's own review.

Note: It says "free for open source", but doesn't say "free forever".

Should we give it a try?
I've sent a request, but since I'm not an owner, it requires your approval, @bquorning.

Disclaimer: I'm not in any way affiliated with the tool, its company or authors.

**Comment by bquorning** [comment]
I don’t know… I find GitHub’s UI pretty ok actually. Are the any particular problems that you want Crocodile to fix?

I am a bit worried that adding a new UI might confuse new contributors. Or maybe we need to manually add access for new contributors?

**Reply by pirj** [reply]
I'll have a look on some personal repo if Croc reviews are forced, or if it's just for reviewers, and it's able to interop with the regular GitHub reviews for contributors.

**Comment by pirj** [comment]
> Crocodile will shutdown in May 2024.
